testsuite/tests/typing-layouts-or-null/test_beta.ml

```diff
@@ -6,7 +6,7 @@
 type t_any_non_null : any_non_null
 
 [%%expect{|
-type t_any_non_null : any_non_null
+type t_any_non_null : any
```

Comment:
I don't know enough about or_null - is this desired?

Reply:
`any` is parsed as `any_non_null` in alpha extensions aren't on, so I think this just changes the displayed name to `any` while still leaving it non-null. I should remove this whole hack, it's bad.

Reply:
This change is fine for now.